src/main/java/com/basho/riak/client/core/query/timeseries/CollectionConverters.java

```diff
@@ -104,16 +110,44 @@ private static FullColumnDescription convertDescribeResultRowToFullColumnDescrip
         final ColumnDescription.ColumnType type =
                 ColumnDescription.ColumnType.valueOf(typeString.toUpperCase(Locale.ENGLISH));
 
+        final Quantum quantum = parseQuantumCells(cells);
+
         return new FullColumnDescription(name,
                                          type,
                                          isNullable,
                                          partitionKeyOrdinal,
-                                         localKeyOrdinal);
+                                         localKeyOrdinal,
+                                         quantum);
+    }
+
+    private static Quantum parseQuantumCells(List<Cell> cells)
+    {
+        final boolean hasQuantum = cells.size() > 5 && cells.get(5) != null && cells.get(6) != null;
```

Comment:
The idea was that we always have a Quantum for format2 (extended describe table), but, yes? since quanta is not mandatory.. it might be a case